### PR TITLE
rramyr's First Goodie

### PR DESCRIPTION
**Pull request by rramyr**
screen shot 2015-04-06 at 1 19 30 am

# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
Query my first awesome goodie

**What problem does your Instant Answer solve (Why is it better than organic links)?**
First DuckDuckHack Goodie tutorial

**What is the data source for your Instant Answer? (Provide a link if possible)**
X

**Why did you choose this data source?**
X

**Are there any other alternative (better) data sources?**
X

**What are some example queries that trigger this Instant Answer?**
duckduckhack githubrramyr

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
X

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Yes, first hack for creating an Instant answer

**Which existing Instant Answers will this one supercede/overlap with?**
Other First Goodie hacks

**Are you having any problems? Do you need our help with anything?**
No

**Where did you hear about DuckDuckHack? (For first time contributors)**
Through AngelList Jobs and followed on Facebook. Using DuckDuckGo search engine and super excited to see the instant answers and looking forward for doing more hacks! Wow, DuckDuckHack is so amazing!

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
GitHubrramyr is awesome and has successfully completed the DuckDuckHack Goodie tutorial!

## Checklist
Please place an 'X' where appropriate.

```
- [] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/GitHubrramyr.pm [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package DDG::Goodie::IsAwesome::GitHubrramyr;
+# ABSTRACT: GitHubrramyr's first Goodie
+# # Start at https://duck.co/duckduckhack/goodie_overview if you are new
+# to instant answer development
+
```
Comment (MrChrisW) [inline lib/DDG/Goodie/IsAwesome/GitHubrramyr.pm]: You can remove comments in line 2 and 4.

**Comment by MrChrisW** [comment]
Hi @rramyr, thanks for submitting your first Goodie! 

I think instances of the words `GitHubUsername` and `GitHubrramyr` are meant to be just `rramyr` ? There's no need to append `GitHub` to your username :smile: 

Let me know if you have any questions

**Comment by rramyr** [comment]
Thanks Chris! I get it now :+1 Updated the files. Let me know if last line in test is a problem.

**Comment by MrChrisW** [comment]
No problem @rramyr I'm happy to help. The test file looks fine! I've left one more comment, once that's updated then this is good to **merge** :smiley:

**Comment by rramyr** [comment]
Thank you! I have added the Abstract comment back :smile:

**Comment by MrChrisW** [comment]
@rramyr Congratulations on making your first Goodie :cake: 

Now that you've made your first Goodie, we really hope you'll continue to contribute and make more Instant Answers! Please checkout our [Ideas forum](https://duck.co/ideas) if you're unsure about what to build. 

There's always lots of discussion in our [GitHub repositories](https://github.com/duckduckgo) feel free to tackle issues or submit pull requests.

**Comment by rramyr** [comment]
Yay!

**Comment by rramyr** [comment]
Hey Chris! how do I look up for the answer once it is merged? Can I see it live?

**Comment by mintsoft** [comment]
@rramyr It should be deployed pretty soon ( a week or so) @moollaza or @jagtalon will reply once it's live to let you know!

**Comment by moollaza** [comment]
@rramyr congrats! It's live: https://duckduckgo.com/?q=duckduckhack+rramyr&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Pull request merged**